stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -64,7 +64,6 @@ end
 
         @testset "throw for non-square input" begin
             A = rand(eltya, 2, 3)
-            @test_throws DimensionMismatch LinearAlgebra.chol!(A)
```

Comment:
(This straggler generated a depwarn, and is now redundant with the test two lines below.)